frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -185,6 +185,54 @@ export class FieldValuesWidget extends Component {
     }
   }
 
+  getSearchableTokenFieldPlaceholder(fields, firstField) {
```

Comment:
When methods like these get moved out component files (this file is > 500 loc), and into little utils files, we will be able to have far simpler names for these methods.

We can make "TokenFieldPlaceholder" implicit in the function names once the file includes that in its own name.